code/addons/docs/src/blocks/components/Preview.tsx

```diff
@@ -252,7 +257,7 @@ export const Preview: FC<PreviewProps> = ({
           border
           zoom={(z: number) => setScale(scale * z)}
           resetZoom={() => setScale(1)}
-          storyId={getStoryId(children)}
+          storyId={getStoryId(getChildProps(children), context)}
```

Comment:
@shilman we might be able to test this with a story that uses a play function to examine the URL of the button. WDYT?